mts-config-server/pom.xml

```diff
@@ -1,23 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>uk.ac.ox.ndph.mts</groupId>
-        <artifactId>mts-services-parent</artifactId>
-        <version>0.0.1-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.4.2</version>
+        <relativePath/> <!-- lookup parent from repository -->
     </parent>
-    <artifactId>sc-config-server</artifactId>
-    <name>sc-config-server</name>
-    <description>Spring Cloud configuration server.
-    </description>
+    <groupId>uk.ac.ox.ndph.mts</groupId>
+    <artifactId>mts-config-server</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>mts-config-server</name>
+    <description>MTS Config Server</description>
```

Comment:
I prefer to revert this file (maybe except the relative path part)

Reply:
If we keep MTS-xx it will be necessary?